docs/api-reference/next/image.md

```diff
@@ -375,49 +451,49 @@ export default function myImageLoader({ src, width, quality }) {
 
 Alternatively, you can use the [`loader` prop](#loader) to configure each instance of `next/image`.
 
-## Advanced
+## Advanced Configuration Options
 
 The following configuration is for advanced use cases and is usually not necessary. If you choose to configure the properties below, you will override any changes to the Next.js defaults in future updates.
 
-### Device Sizes
+### `deviceSizes`
 
 If you know the expected device widths of your users, you can specify a list of device width breakpoints using the `deviceSizes` property in `next.config.js`. These widths are used when the `next/image` component uses [`sizes`](#sizes) prop to ensure the correct image is served for user's device.
 
 If no configuration is provided, the default below is used.
 
-```js
+```jsx
 module.exports = {
   images: {
     deviceSizes: [640, 750, 828, 1080, 1200, 1920, 2048, 3840],
   },
 }
 ```
 
-### Image Sizes
+### `imageSizes`
 
-You can specify a list of image widths using the `images.imageSizes` property in your `next.config.js` file. These widths are concatenated with the array of [device sizes](#device-sizes) to form the full array of sizes used to generate image [srcset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset)s.
+You can specify a list of image widths using the `images.imageSizes` property in your `next.config.js` file. These widths are concatenated with the array of [device sizes](#devicesizes) to form the full array of sizes used to generate image [srcsets](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset).
 
 The reason there are two separate lists is that imageSizes is only used for images which provide a [`sizes`](#sizes) prop, which indicates that the image is less than the full width of the screen. **Therefore, the sizes in imageSizes should all be smaller than the smallest size in deviceSizes.**
 
 If no configuration is provided, the default below is used.
 
-```js
+```jsx
 module.exports = {
   images: {
     imageSizes: [16, 32, 48, 64, 96, 128, 256, 384],
   },
 }
 ```
 
-### Acceptable Formats
+### `formats`
```

Comment:
Can we call this "Formats Configuration"?

The reason I bring this up is that the docs will be broken if we add a `formats` prop in the future since it wont know if we are linking to the prop or config.

I think all configuration should have it in the title.

Reply:
Although we already have this problem with `unoptimized` which can be a prop or config. If they end up have the same value type then its okay but if not, it could be a problem.